app/views/questionnaire.py

```diff
@@ -308,7 +307,7 @@ def get_view_submission(schema, eq_id, form_type):  # pylint: disable=unused-arg
             encrypter = StorageEncryption(current_user.user_id, current_user.user_ik, pepper)
 
             submitted_data = json.loads(encrypter.decrypt_data(submitted_data.data))
-            answer_store = AnswerStore(existing_answers=submitted_data.get('answers'))
+            answer_store = AnswerStore(answer_map=submitted_data.get('answers'))
```

Comment:
We'll need this to deal with the old answer store format for a short period of time after deploying these changes.